packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -844,6 +844,8 @@ const ruleset$1 = {
             description: "Operation should have a summary or description.",
             message: "Operation should have a summary or description.",
             severity: "warn",
+            disableForTypeSpec: true,
```

Comment:
While I'm fine with hardcoding these configurations for now I do think we need to work towards a goal of having these configurations be external of the tool so we don't need to release a new tool to enable or disable specific rules.

Reply:
Wondering if that would not be beneficial to do right now as it seems rulesets are all redefining the same disableForTypeSpec and reason, maybe it would be better to hard code a map for now so we can write the reason once.

Reply:
@weshaggard @timotheeguerin out of scope for now - I need to deliver this by Thursday. I am happy to work on any improvements in a follow-up PR.